app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::PostsController < ApiController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.order(created_at: :desc)
+
+    render json: @posts
+  end
+end
```

Comment:
## [OPTIONAL]

- [ ] As you have already the login authentication logic, I think it would be nice if you protect these routes(`before_action :authorize_request`) that are getting post info.